docs/comparison.md

```diff
@@ -0,0 +1,25 @@
+|   | OpenVoiceOS<br>(Buildroot) | OpenVoiceOS<br>(Manjaro) | Neon AI | Mycroft Mark II<br>(Dinkum) |
+|---|---|---|---|---|
+| Hardware - Compatibility |
+|---|---|---|---|---|
+| Raspberry Pi | 3/3b/3b & 4 | 4 | 4 | Mark II (only) |
+| X86_64 | *planned | No | Unknown | No |
+| Virtual Appliance | *planned | No | Unknown | No |
+| Docker | No<br>*possible | Yes | Yes | No |
+| Hardware - Peripherals |
+| ReSpeaker |   |   |   |   |
+| USB |   |   |   |   |
+| SJ-201 |   |   |   |   |
+| Network Setup - Options |
+| Mobile WiFi Setup<br>Easy device "hotspot" to connect to preset network from phone or pad.  | Yes | No | No | Yes |
+| On device WiFi Setup<br>Configure the WiFi connection on the device itself | Yes | Yes | WIP | No |
+| On screen keyboard | Yes | Yes | No | No |
+| Configuration - Option |
+| Data privacy | Yes | Yes | Yes | Partial |
+| Offline mode | Yes | Yes | Yes | No |
+| Color theming | Yes | Yes | Yes | No |
+| Non-Pairing mode | Yes | Yes | Yes | No |
+| API Acces w/o pairing | Yes<br>most skills | Yes<br>most skills | Yes | No |
```

Comment:
isnt it all skills ?

Reply:
Nope, still working on it. Just pushed for backup purposes and to quickly inspect if i got that MD table markup right.

Reply:
Oh hang on, realise what you meant. Had that question as well. Just copied it over from the neon table for now.